molecule/default/prepare.yml

```
@@ -1,8 +1,21 @@
 ---
 - name: Prepare
   hosts: all
-  tasks:
-
+  vars:
+    mysql_root_username: root
+    mysql_root_password: netways0815!
+    mysql_databases:
+      - name: testDatabase
+        encoding: utf8
+        collation: utf8_general_ci
+        replicate: 0
+    mysql_users: 
+      - name: testDatabase
+        host: 127.0.0.1
+        password: netways0815!
```

```suggestion
        password: ChangeMe
```

Please use a more common password for tests